_docs-sources/foundations/landing-zone/manage-accounts.md

```diff
@@ -35,35 +35,95 @@ Below you'll find a table with common AWS account operations and the Gruntwork r
 
 ### Remove an AWS account
 
-This operation removes an AWS account from AWS Control Tower but does _not_ delete the AWS account.
+We recommend using AWS Organizations interface for closing an account and destroying all of its resources for a smooth experience due to current flakiness of the AWS Service Catalog when performing deletes via OpenTofu/Terraform providers. See the steps below for the complete sequence of actions to remove an account.
 
-1. Create a branch and delete the directory in your `infrastructure-live` repository that corresponds to the account you would like to remove.
+:::note
+AWS Organizations will permanently [close an account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-closing.html) after 90 days.
+:::
 
-    :::warning
-    This will delete all resources in the account. Make sure you are deleting the correct directory.
-    :::
 
-    Push your changes and create a PR. Pipelines will detect that an account is deleted. Verify that all expected resources will be removed in the `plan` output.
 
-    Once you have confirmed everything looks as expected, merge the PR.
+#### 1. (Optional) Destroy resources in the account to be removed
+
+You may choose or need to destroy some or all of your provisioned resources, if the resources are managed by Terragrunt in your infrastructure repository which has Gruntwork Pipelines integrated, you can destroy them by doing the following:
+
+1. Checkout a new branch in the infrastructure repository
+1. Remove the folder(s) containing the resources you want to destroy
+1. Commit your changes and create a Pull Request which will trigger Gruntwork Pipelines to destroy the resource once the Pull Request is merged.
```

Comment:
I wonder if we should call out that they should avoid deleting any roles or the OIDC provider here so that they don't accidentally remove their ability to complete deletions.